test/unit/server-ingress.bats

```diff
@@ -131,6 +131,18 @@ load _helpers
   [ "${actual}" = "nginx" ]
 }
 
+@test "server/ingress: ingressClassName added to object spec - string" {
```

Comment:
It would also be good to add the opposite test, that ingressClassName is not added to the Ingress spec when server.ingress.ingressClassName is not specified.